.conform.yaml

```diff
@@ -3,11 +3,11 @@ policies:
   - type: commit
     spec:
       header:
-        length: 80
+        length: 75
         imperative: false
         invalidLastCharacters: .
       body:
```

Comment:
Can the length be specified separately for body? There is also a limit for it, and it used to be 72 (see e.g. https://review.coreboot.org/c/coreboot/+/82041/8..9//COMMIT_MSG#b12), I don't know if it changed recently or not.

Reply:
Seems to be new: https://review.coreboot.org/c/coreboot/+/85915

Reply:
At present, the body is just checked if it exists. There is no enforcement on the line length whatsoever, it seems.